### PR TITLE
Adds support to Laravel 5.8

### PR DESCRIPTION
**Pull request by KaneCohen**

### CODE REVIEW

**Comment by stephandesouza** [comment]
@vinicius73 could you help us? 🍺

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "artesaos/seotools",
+  "name": "cohensive/seotools",
```
Comment (routegroup) [inline composer.json]: Don't change it

Comment (routegroup) [reply composer.json]: Btw. you should make it on different branch than master.

Comment (KaneCohen) [reply composer.json]: Right. Forgot I made the PR. Had to push new package while waiting for PR to be merged.

**Comment by thehelvetian** [comment]
While you're at it please remove support for `5.0`:

```
laravel/framework 5.0.x-dev requires ext-mcrypt * -> the requested PHP extension mcrypt is missing from your system.
```

**Comment by KaneCohen** [comment]
@thehelvetian i'll leave that up to maintainer to decide. `ext-mcrypt` could be installed by the user if necessary.

**Comment by vbezruchkin** [comment]
Hey there, vote for 5.8 support :) Please merge. Thanks

**Comment by connor11528** [comment]
Since this is not merged in yet, how can we use this branch in our projects? I am trying to upgrade to 5.8 and cannot because of this issue. I'd like to install this branch. Is it available on composer or must I do something locally? @KaneCohen

**Comment by KaneCohen** [comment]
@connor11528 add this to your `composer.json`:

```json
"repositories": [
  {
    "type": "vcs",
    "url": "https://github.com/kanecohen/seotools"
  }
]
```
and in `require` part of composer add `"cohensive/seotools": "dev-master"`

**Comment by connor11528** [comment]
@KaneCohen getting an error: 

```
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package cohensive/seotools could not be found in any version, there may be a typo in the package name.
```


I changed it to `"artesaos/seotools": "dev-master",` ran `composer dump-autoload`, changed to `"laravel/framework": "5.8.3",` and then `composer update` worked. Thank you!

**Comment by emtudo** [comment]
Thanks!

**Pull request merged**